reconcile/utils/gitlab_api.py

```diff
@@ -217,10 +231,12 @@ def add_group_member(self, group_name, username, access):
         if not self.check_group_exists(group_name):
             logging.error(group_name + " group not found")
         else:
+            gitlab_request.labels(integration=INTEGRATION_NAME).inc()
             group = self.gl.groups.get(group_name)
             user = self.get_user(username)
             access_level = self.get_access_level(access)
             if user is not None:
+                gitlab_request.labels(integration=INTEGRATION_NAME).inc()
```

Comment:
It looks like there's another call to `group.members.get` below in the exception handling?

Reply:
ah yes! I missed that one.